app/fruits/[fruitName]/actions.js

```diff
@@ -0,0 +1,26 @@
+'use server';
+
+import { cookies } from 'next/headers';
+import { getCookie } from '../../../util/cookies';
+import { parseJson } from '../../../util/json';
+
+export async function createComment(fruitId, comment) {
+  const fruitComments = parseJson(getCookie('fruitComments'));
+
+  const currentComments = Array.isArray(fruitComments)
```

Comment:
- `currentComments` is a bit confusing of a name to me
- if you use the method below, you don't need the extra variable

Reply:
[b2409c5](https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/pull/15/commits/5f5e19ebf8f50871e0209fc5f4d835a662d23153)